added purchase link

RAJU2529 opened the pull request:
as per user feedback #7251 , so i added project plans and visio plans links. added highlights of Project plans and visio  plans.

RAJU2529 committed:
added purchase link

as per user feedback #7251 , so i added project plans and visio plans links. added highlights of Project plans and visio  plans.

denisebmsft committed:
Merge branch 'public' into patch-8

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

RAJU2529 committed:
Merge branch 'public' into patch-8

JohanFreelancer9 commented on microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md:
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
+
+• Choose from dozens of professional templates, starter diagrams, and shapes.
+
+• Access and edit Visio files directly inside Microsoft Teams.*
+
+• Embed Visio diagrams into Power BI reports for unified perspectives.*
```
```suggestion
• Access and edit Visio files directly inside Microsoft Teams.

• Embed Visio diagrams into Power BI reports for unified perspectives.
```

JohanFreelancer9 commented on microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md:
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
+
+• Enjoy around-the-clock support from Microsoft.
+
+## Purchase Project and Visio Plans through Microsoft 365 link
```
```suggestion
## Microsoft 365 links to purchase Project and Visio plans
```

JohanFreelancer9 commented on microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md:
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
+
+## Purchase Project and Visio Plans through Microsoft 365 link
+
+> **Project Plan 1** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-1?activetab=pivot:overviewtab)
+> 
+> **Project Plan 3** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-3?activetab=pivot:overviewtab)
+> 
+> **Visio Plan 1** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-1?activetab=pivot:overviewtab)
+> 
+> **Visio Plan 2** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-2?activetab=pivot:overviewtab)
+> 
```
```suggestion
**Project Plan 1** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-1?activetab=pivot:overviewtab).

**Project Plan 3** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-3?activetab=pivot:overviewtab).

**Visio Plan 1** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-1?activetab=pivot:overviewtab).

**Visio Plan 2** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-2?activetab=pivot:overviewtab).
```

RAJU2529 committed:
Update microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

Accepted

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

RAJU2529 committed:
Update microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

Accepted

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

RAJU2529 committed:
Update microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

Accepted

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

JohanFreelancer9 reviewed (changes requested):
Copy review.

yogkumgit commented:
@cmcatee-MSFT Please merge this PR. Thanks

RAJU2529 committed:
Merge branch 'public' into patch-8

cmcatee-MSFT committed:
Update allowselfservicepurchase-powershell.md

No new sections needed. Moved up the note the PM added to the wrong section. Problem solved.

cmcatee-MSFT committed:
Merge branch 'public' into patch-8

RAJU2529 reviewed (commented):
Please merge it

cmcatee-MSFT committed:
Merge branch 'public' into patch-8
